### DISCUSSION TITLE
uWS::Loop::defer() not firing.

### DISCUSSION THREAD

**Discussion by jacobsologub** [uNetworking/uWebSockets#1596] [Q&A] [answered]
`uWS::Loop::defer()` doesn't seem to be firing...

I've got a separate thread firing at about 1 seconds interval(s).
```
uWS::Loop::get()->defer ([] () {
    auto x = 0; // not fired.
});
```
Loop::deferQueues just keeps building up.

I'm on 8410f7e

**Comment by uNetworkingAB** [comment, marked as answer]
You are getting a new loop for the queueing thread. You need to grab the loop of the target thread. Loop::get() is thread local

**Comment by jacobsologub** [comment]
@uNetworkingAB Yeah just sorted that out. Thanks.
